worlds/sm64ex/__init__.py

```diff
@@ -71,17 +102,8 @@ def create_item(self, name: str) -> Item:
 
         return item
 
-    def create_items(self):
-        starcount = self.multiworld.AmountOfStars[self.player].value
-        if (not self.multiworld.EnableCoinStars[self.player].value):
-            starcount = max(35,self.multiworld.AmountOfStars[self.player].value-15)
-        starcount = max(starcount, self.multiworld.FirstBowserStarDoorCost[self.player].value,
-                        self.multiworld.BasementStarDoorCost[self.player].value, self.multiworld.SecondFloorStarDoorCost[self.player].value,
-                        self.multiworld.MIPS1Cost[self.player].value, self.multiworld.MIPS2Cost[self.player].value,
-                        self.multiworld.StarsToFinish[self.player].value)
-        self.multiworld.itempool += [self.create_item("Power Star") for i in range(0,starcount)]
-        self.multiworld.itempool += [self.create_item("1Up Mushroom") for i in range(starcount,120 - (15 if not self.multiworld.EnableCoinStars[self.player].value else 0))]
```

Comment:
Items must be created in `create_items`, per #1460

Reply:
Making Starcraft 2 comply with that PR is going to be involve rewriting the whole codebase from start-to-finish.  A lot easier to fix it in SM64.  Done.  (https://github.com/ArchipelagoMW/Archipelago/pull/2569/commits/163a61c6f62ec64009df4af4749dc036b3b54773)

Reply:
Just to be a bit pedantic: It must be done *by* create_items, not *in* create_items - If for some reason you want to make items in generate_early, you can do that

Reply:
> I think you are no longer creating the ExclamationBox items

Why would they not be created?  I haven't done much to change the code in that part.

I cannot create the cannons or exclamation box items in create_items if the options result in them getting hard-placed later, as that would be modifying the item pool after create_items.

Reply:
My bad, misread